ROLES.md

```diff
@@ -242,7 +274,89 @@ The following apply to the area / component for which one would be an owner.
 
   - Fixing bugs.
 
-  - Mentoring and guiding approvers, members, and contributors.
+  - Identifying needed enhancements / areas for improvement / etc.
+
+  - Execute pay-down of technical debt.
+
+- Mentoring and guiding approvers, members, and new contributors.
+
+## Working Group Execution Lead
+
+Working group execution leads, or just ‘execution leads’, are responsible for
+the overall health and execution of the working group itself. Execution leads
+work with tech leads to ensure that the working group is making progress toward
+its goals, is aligned with the project roadmap, etc. The execution lead may also
+be the tech lead in a smaller working group, but they are distinct roles.
+
+### Requirements
+
+- Participant in the working group for at least 3 months, for example as scribe
+  or approver.
+
+- Recognized as having expertise in the group’s subject matter.
+
+- Sponsored by the technical oversight committee.
+
+### Responsibilities and privileges
+
+- Run their working group as explained in the
+  [Working Group Processes](./mechanics/WORKING-GROUP-PROCESSES.md).
+
+  - Meetings. Prepare the agenda and run the regular working group meetings.
+
+  - Notes. Ensure that meeting notes are kept up to date. Provide a link to the
+    recorded meeting in the notes. The lead may delegate note-taking duties to
+    the scribe.
+
+  - Roadmap. Establish and maintain a roadmap for the working group outlining
+    the areas of focus for the working group over the next 3 months.
+
+  - Report. Report current status to the TOC meeting every 6 weeks.
+
+- Holistic responsibility for their working group's [feature
+  tracks](./mechanics/FEATURE-TRACKS.md): tracking, health, and execution.
```

Comment:
I wonder if we should have a specific call out for helping to manage cross-WG execution, incl. but not limited to how a WG is represented in horizontal efforts?

Reply:
I think that would be useful, maybe we should also discuss that in a joint toc/steering session?